src/quo/components/avatars/group_avatar/view.cljs

```diff
@@ -50,14 +49,6 @@
             {:size  :paragraph-1
              :style (dissoc (style/avatar-identifier theme) :font-size)}
             emoji])
-         chat-name
```

Comment:
group avatar don't support initials
https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=179-9760&mode=design&t=pV0pn0UiszwVIz2z-0

Reply:
I think the same component was used for community channels too, That's why it had initials.